packages/discord.js/src/structures/CommandInteraction.js

```diff
@@ -204,6 +204,7 @@ class CommandInteraction extends Interaction {
   deleteReply() {}
   followUp() {}
   showModal() {}
+  awaitModal() {}
```

Comment:
awaitModal or awaitModalSubmit?

```suggestion
  awaitModalSubmit() {}
```